### PR TITLE
fix/harmonize_azure_fields

### PR DESCRIPTION
**Pull request by clement-burtscher-sekoia**
## Description

Fixes https://github.com/SekoiaLab/integration/issues/1579

### Added

- Added `azuread.properties` aliases matching existing `azure.entraid.properties` mappings:
    - `azuread.properties.appId`
    - `azuread.properties.conditionalAccessStatus`
    - `azuread.properties.resourceId`
    - `azuread.properties.riskDetail`
    - `azuread.properties.riskEventTypes`
    - `azuread.properties.riskLevelAggregated`
    - `azuread.properties.riskLevelDuringSignIn`
    - `azuread.properties.riskState`
    - `azuread.properties.targetServicePrincipalDisplayName`

## Type of Change

- [ ] New intake format
- [x] Update to existing intake format
- [x] Bug fix
- [ ] Documentation update
- [ ] Other (please describe):

## Affected Intake Formats

- Vendor/product:

### 🔒 CRITICAL - Security Requirements (MUST BE VERIFIED FIRST)

- [ ] **No real email addresses** in test files (use: user@example.com, test@example.org)
- [ ] **No real passwords or API keys** (use: "P@ssw0rd!", "xxxxxxxxxxxx", "FAKE_TOKEN_123")
- [ ] **No real IP addresses** (use TEST-NET ranges: 192.0.2.x, 198.51.100.x, 203.0.113.x)
- [ ] **No real phone numbers** (use: +1-555-0100 or similar test numbers)
- [ ] **No real names or PII** (use: "John Doe", "Jane Smith", "User123")
- [ ] **No real company/organization names** in test data (unless public vendors)
- [ ] **No real usernames, hostnames, or domain names** (use: user1, host.example.com, test.corp)
- [ ] All test data has been anonymized and verified

### Required for All PRs

- [ ] Code has been linted
    - [ ] with the linter for manifest, smart-descriptions, and test files (`uv run python linter.py check --changes` in utils/)
    - [ ] with Prettier for the parser code files (`npx prettier --write <module>/<format>/ingest/parser.yml`)
- [ ] All CI/CD checks pass
- [ ] Changes follow the contribution guidelines in `CONTRIBUTING.md`

### Required for New/Updated Intake Formats

- [ ] Clear descriptions provided for modules and formats
- [ ] Logo files included for new modules/formats
- [ ] Parser test coverage is at least 75%
- [ ] At least one smart-description is provided
- [ ] README.md updated (if applicable)
- [ ] Taxonomy mappings are correct and documented
- [ ] Test cases cover edge cases and error handling

### Testing

- [ ] Local tests pass (`uv run pytest` in utils/)
- [ ] Sample data is representative and **completely anonymized**
- [ ] **No sensitive information** (PII, credentials, real IPs, real emails) in test files
- [ ] Test data uses example.com/example.org domains and TEST-NET IP ranges
- [ ] Parser handles malformed data gracefully

### Documentation

- [ ] Code changes are documented
- [ ] Smart-descriptions are clear and informative
- [ ] Field mappings are explained

## Related Issues

Closes #

## Summary by Sourcery

Align Azure AD field aliases with existing Entra ID properties in the Microsoft Entra ID Graph API intake format.

New Features:
- Introduce azuread.properties field definitions mirroring existing azure.entraid.properties mappings.

Enhancements:
- Map azuread.properties fields in the parser to populate them from Graph API sign-in and audit event payloads.
- Document the added azuread.properties aliases in the module changelog.

### CODE REVIEW

File: Microsoft/microsoft-entraid-graphapi/CHANGELOG.md [hunk 1]
```diff
@@ -6,3 +6,18 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [Unreleased] - 2026-05-27
+
```
[the remark this answers is not in the document]
Comment (clement-burtscher-sekoia) [reply Microsoft/microsoft-entraid-graphapi/CHANGELOG.md:9]: Fixed with commit [Clean changelog](https://github.com/SEKOIA-IO/intake-formats/pull/2561/changes/9ed218870230bbe0ae8e945d3996e864880b95cb)

File: Microsoft/microsoft-entraid-graphapi/CHANGELOG.md [hunk 2]
```diff
@@ -6,3 +6,18 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [Unreleased] - 2026-05-27
```
[the remark this answers is not in the document]
Comment (clement-burtscher-sekoia) [reply Microsoft/microsoft-entraid-graphapi/CHANGELOG.md:8]: Fixed with commit [Clean changelog](https://github.com/SEKOIA-IO/intake-formats/pull/2561/changes/9ed218870230bbe0ae8e945d3996e864880b95cb)

**Review by LenaigKaliou — DISMISSED** [review]
LGTM

**Review by TOUFIKIzakarya — APPROVED** [review]
Thanks for the improvement.
LGFM
